apps/wallet-mobile/src/features/Swap/common/useSwapTx.ts

```diff
@@ -28,7 +28,8 @@ export const useSwapTx = (options?: UseMutationOptions<YoroiUnsignedTx, Error, {
 
   const {wallet, meta} = useSelectedWallet()
   const mutation = useMutationWithInvalidations({
-    mutationFn: (data) => wallet.createUnsignedTx({entries: data.entries, metadata, addressMode: meta.addressMode}),
```

Comment:
Do we know since when `createUnsignedTx` expects `auxiliaryData` instead of `metadata`?

Reply:
the wallet class has changed a lot recently so maybe it changed and we didn't update that function accordingly

Reply:
Typescript should have complained that the function inside the wallet class doesn't have the same arguments as the type.

Reply:
yap the issue is since both are optionals, it looks like TS (linter) at least, just disregard them, @banklesss I think it should be `metadata` since `auxiliaryData` was the old name for the tx metadatum, update the ledger part, and make sure that all is passing as `metadata` please.

Reply:
@stackchain i just found the ledger `auxiliaryData` is needed because this [type](https://github.com/cardano-foundation/ledgerjs-hw-app-cardano/blob/ee543694deaa9dd39d049d3efb888515bf058faa/src/types/public.ts#L1705) 

needed [here](https://github.com/Emurgo/yoroi/blob/develop/apps/wallet-mobile/src/features/Discover/common/ledger.ts#L201)